maestro-client/src/main/java/maestro/utils/StringUtils.kt

```diff
@@ -12,4 +12,7 @@ object StringUtils {
         }
     }
 
+    fun String.toSanitizedIOSLink(): String {
+        return this.replace("?", "\\?")
```

Comment:
@artem888 is this correct we need to escape ? when using the xcrun simctl open URL command?

I was getting: 

```
zsh: no matches found: testapp://testpage?testparam={TEST:test}
```

Also do we need to escape something else as well?

Reply:
I tested locally , looks like any special character needs to be sanitized, not only `?`.

smth like that should work:

```
fun sanitizeURL(url: String): String {
    val sanitizedURL = StringBuilder()
    for (ch in url) {
        if (ch.isLetterOrDigit() || ch == '.' || ch == '-' || ch == '_' || ch == '*' || ch == '/') {
            sanitizedURL.append(ch)
        } else {
            sanitizedURL.append(URLEncoder.encode(ch.toString(), "UTF-8"))
        }
    }
    return sanitizedURL.toString()
}
```